adapters/network.js

```diff
@@ -4,75 +4,144 @@ const http = require("http");
 const https = require("https");
 const axios = require("axios");
 const _ = require("lodash");
+const log = require("../logger");
+const { removeUndefinedAndNullValues } = require("../v0/util/index");
 
-const send = async options => {
-  // here the options argument K-Vs will take priority over requestOptions
-  const requestOptions = {
-    ...{
-      // `method` is the request method to be used when making the request
-      method: "get",
+const agentArgs = removeUndefinedAndNullValues({
+  keepAlive: true,
+  maxSockets: Number.isNaN(
+    Number.parseInt(process.env.NETWORK_AGENT_MAX_SOCKET, 10)
```

Comment:
Should we still have `maxSockets`/`maxFreeSockets`/`keepAlive` to be set ?

Reply:
Since we moved this config via transformer env vars and we are using this function for transformer api calls (example Marketo bulk upload) so kept it for that use case

Reply:
I would say, let's clear this out now & add it at a later point of time, if there is a requirement as such